src/index.ts

```diff
@@ -95,7 +103,7 @@ export type ConstructorParams<Actual> = Actual extends new (...args: infer P) =>
 
 type MismatchArgs<B extends boolean, C extends boolean> = Eq<B, C> extends true ? [] : [never]
 
-export interface ExpectTypeOf<Actual, B extends boolean> {
+export interface ExpectTypeOf<Actual, B extends boolean, Branded = false> {
```

Comment:
Maybe we should merge in a separate PR that renames `B` to `Positive` or something? I feel like I've seen that elsewhere on one of the PRs recently, am I remembering that right?

Reply:
I'm guessing you're thinking of https://github.com/mmkal/expect-type/pull/13

In that one, I renamed it to `ExpectTypeOf<Actual, ExpectedResult extends boolean>`

I could incorporate parts of that PR to just the generic renaming.